network/utils/network.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package utils
+
+import (
+	"github.com/aws/amazon-vpc-cni-plugins/network/config"
+	"github.com/aws/amazon-vpc-cni-plugins/network/eni"
+	"github.com/aws/amazon-vpc-cni-plugins/network/vpc"
+
+	log "github.com/cihub/seelog"
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+)
+
+// CreateTAPLink creates a TAP link and a bridge in the target network namespace.
+// It also sets the bridge as the master of the TAP link and the ENI link that is
+// passed to it so that traffic may flow from the ENI to the TAP link.
+func CreateTAPLink(
```

Comment:
Should we call this `CreateTAPBridge` then?

Reply:
Changed in next revision.